src/js/components/cycleCount/tableCell/QuantityCell.jsx

```diff
@@ -54,19 +73,28 @@ const QuantityCell = ({
         value,
       }),
     );
+    onBlurValidationHandler();
   };
 
   return (
     <TableCell className="rt-td rt-td-count-step">
       <InputField
         type="number"
         value={value}
+        showErrorBorder={shouldShowError}
         onChange={onChange}
         onBlur={onBlur}
         className="m-1 w-75"
         hideErrorMessageWrapper
         min="0"
       />
+      {shouldShowError && (
+        <CustomTooltip
+          content={error}
+          className="tooltip-icon tooltip-icon--error"
+          icon={RiErrorWarningLine}
+        />
+      )}
```

Comment:
it's be great if this logic could exist within the InputField itself (or maybe in a smarter wrapper component like ValidatedInputField) so that we can standardize our error tooltips across the apps. IMO the ideal would be that we just pass errorTriggers=shouldShowError and errorMessage=error fields to the component and it'd handle the tooltips internally.

the way you have it is fine for now though